pkgs/development/compilers/ghc/8.2.2.nix

```diff
@@ -239,4 +239,8 @@ stdenv.mkDerivation rec {
     inherit (ghc.meta) license platforms;
   };
 
-}
+} // stdenv.lib.optionalAttrs targetPlatform.useAndroidPrebuilt {
+  dontStrip = true;
```

Comment:
Sounds okay for now, but eventually we should try to track down what is happening here. It must be a bug in binutils & patchelf, right? In theory android should just be using standard ELFs.

Reply:
I think it's simpler/stupider than that: a problem from using prebuilt tools. I think if we used the prebuilt tools for `gccCrossStageStatic` and then built our own stage final GCC it would work fine.

Reply:
Ok makes sense. It's surprising that the prebuilt tools would not work though? At least they shouldn't break stuff. Since this is probably needed everywhere, is there a way to set this globally for all prebuilt android?

Reply:
Well I take it back; I actually do remember a strip bug with the wrong strip mangling the binary. My newer setup hook included a largely untested provision to do one or the other strip, but if they 

There is a bunch of `dontStrip` for cross scattered about that i don't think has been systematically investigated.